.github/workflows/matrix-tests.yml

```diff
@@ -14,7 +15,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v4
-      - uses: FedericoCarboni/setup-ffmpeg@v3
+      - uses: FedericoCarboni/setup-ffmpeg@v2
```

Comment:
why?

Reply:
I left a comment in the commit, but I guess that's not enough:
https://github.com/ReadAlongs/Studio/pull/229/commits/93d68daf8a217f58aa1bb505b0dfdb13266868f6
The reason is we regularly have failed CI runs that fail on downloading ffmpeg. Going back to v2 gets ffmpeg internally from github, so it'll be a lot more reliable, at the fairly minor cost of not being the latest.